crates/virtio-queue/src/defs.rs

```diff
@@ -4,20 +4,20 @@
 
 //! Virtio queue related constant definitions
 
-/// Marks a buffer as continuing via the next field.
-pub const VIRTQ_DESC_F_NEXT: u16 = 0x1;
+// /// Marks a buffer as continuing via the next field.
+// pub const VIRTQ_DESC_F_NEXT: u16 = 0x1;
 
-/// Marks a buffer as device write-only.
-pub const VIRTQ_DESC_F_WRITE: u16 = 0x2;
+// /// Marks a buffer as device write-only.
+// pub const VIRTQ_DESC_F_WRITE: u16 = 0x2;
 
-/// Marks a buffer as containing a list of buffer descriptors.
-pub const VIRTQ_DESC_F_INDIRECT: u16 = 0x4;
+// /// Marks a buffer as containing a list of buffer descriptors.
+// pub const VIRTQ_DESC_F_INDIRECT: u16 = 0x4;
 
-/// Flag to disable guest notification for used descriptors.
-pub const VIRTQ_USED_F_NO_NOTIFY: u16 = 0x1;
+// /// Flag to disable guest notification for used descriptors.
+// pub const VIRTQ_USED_F_NO_NOTIFY: u16 = 0x1;
 
-/// Size of one element in the used ring, id (le32) + len (le32).
-pub(crate) const VIRTQ_USED_ELEMENT_SIZE: u64 = 8;
+// /// Size of one element in the used ring, id (le32) + len (le32).
+// pub(crate) const VIRTQ_USED_ELEMENT_SIZE: u64 = 8;
```

Comment:
The commented code should be deleted.

Reply:
@uran0sH there are multiple places with commented code. Can you address them all? I didn't leave a comment for all of them.

Reply:
I think I have deleted them all.